cookbooks/solr/recipes/default.rb

```diff
@@ -0,0 +1,140 @@
+#
+# Cookbook Name:: solr
+# Recipe:: default
+#
+# We specify what version we want below.
+use_default_java = false
+java_package_name = "dev-java/icedtea-bin"
+java_version = node['solr']['java_version']
+java_eselect_version = node['solr']['java_eselect_version']
+solr_version = node['solr']['solr_version']
+solr_file = "solr-#{solr_version}.tgz"
+solr_url = "https://archive.apache.org/dist/lucene/solr/#{solr_version}/#{solr_file}"
+core_name = node['solr']['core_name']
+
+username = node['dna']['users'].first['username']
+
+# Install Solr
+if ('solo' == node['dna']['instance_role'])  ||
```

Comment:
This line should be customizable. Rethink this so that customers can choose where to install solr by just modifying custom-solr.